packages/telepathy_logger.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Telepathy_logger < Package
+  description 'A library for GLib-based Telepathy components.'
+  homepage 'https://telepathy.freedesktop.org/'
+  version '0.8.2'
+  compatibility 'all'
+  source_url 'https://telepathy.freedesktop.org/releases/telepathy-logger/telepathy-logger-0.8.2.tar.bz2'
+  source_sha256 '8fcad534d653b1b365132c5b158adae947810ffbae9843f72dd1797966415dae'
+
+  depends_on 'dbus_glib'
+  depends_on 'libx11'
+  depends_on 'gobject_introspection'
+  depends_on 'telepathy_glib'
+  
+  def self.build
+    system "env PYTHON=python2.7 \
```

Comment:
Can we use python 3?

Reply:
package file attached above uses python3 and is patched for such.

Reply:
Actually, I take it back. telepathy logger is both broken and abandoned, as far as I can tell, and I can't get it to compile properly.

Reply:
that's what I was noticing, maybe there are some patches for it on salsa.debian.org. there usually are.